packages/core/src/index.ts

```diff
@@ -1,12 +1,13 @@
 import {
   jsx as emotion,
   ThemeContext as EmotionContext,
-  InterpolationWithTheme,
-} from '@emotion/core'
+  Interpolation,
+} from '@emotion/react'
 import { css, Theme } from '@theme-ui/css'
 import * as React from 'react'
 import deepmerge from 'deepmerge'
-import packageInfo from '@emotion/core/package.json'
+import packageInfo from '@emotion/react/package.json'
+import {} from '@emotion/react/types/css-prop'
```

Comment:
Let's leave this a side note here. I think we should also make `sx` prop types opt in, as we have users who may prefer to use `@theme-ui/components` instead of JSX pragma.